src/components/Reporta.tsx

```diff
@@ -1,16 +1,19 @@
+import { useCallback } from "react";
 import styles from "./Reporta.module.css";
 
 export default function ReportaForm() {
-  function Submit(e) {
-    const formEle = document.querySelector("#formSheet");
+  const submit: React.FormEventHandler<HTMLFormElement> = useCallback((e) => {
+    const formEle = document.querySelector<HTMLFormElement>("#formSheet");
     e.preventDefault;
```

Comment:
these seem like confusing notations,
can't it be:
const formEle : HTMLFormElement = document.querySelector() ?

Reply:
My explanation was wrong! talked it out in group chat and pushed `const formEle : HTMLFormElement | null`